### DISCUSSION TITLE
reuse last entered value for vw_qgep_reach

### DISCUSSION THREAD

**Discussion by urskaufmann** [QGEP/QGEP#735] [Q&A]
In QGIS 3.22 this reuse can be definied individually for every field. For vw_qgep_reach, this reuse is fix programmed in the qgep-code for the fields of the general tab. 
Is it possible to change this, that also for vw_qgep_reach the individual definition one each field can be used?

**Comment by sjib** [comment]
@3nids Can you give an answer to this question?

Here is the view definition for vw_qgep_reach: https://github.com/QGEP/datamodel/blob/master/view/vw_qgep_reach.py
